syntax = "proto3";

package PFDESolver;

import "config.proto";
import "matrix.proto";

message TResult {
    TSolverConfig Config = 1;
    TMatrix Field = 2;
    optional TMatrix SolveMatrix = 3;
    optional TMatrix RealSolution = 4;
    optional string RealSolutionName = 5;
}